libass/x86/rasterizer.asm

```diff
@@ -488,7 +488,7 @@ endstruc
 ;------------------------------------------------------------------------------
 
 %macro GET_RES_ADDR 1
-%if mmsize <= 16 && HAVE_ALIGNED_STACK
+%if mmsize <= 16 && STACK_ALIGNMENT == 16
```

Comment:
To correctly support higher stack alignments that should be
```asm
%if mmsize <= STACK_ALIGNMENT
```

Reply:
Right, sorry. Fixed.